samples/nextjs/jss-create.js

```diff
@@ -17,5 +20,83 @@ module.exports = function createJssProject(argv, nextSteps) {
 
   applyNameToProject(__dirname, argv.name, argv.hostName, 'JssNextWeb');
 
+  if (!argv.fetchMode || !argv.renderMode) {
```

Comment:
Since we want to get away from all the "modes" in general, maybe these variables should be named based on what they change?
For example, if the default render mode is SSG, then the param could be a boolean 'useSSR'

Reply:
Right... the "modes" avoidance... I had considered a flag, but we lose some flexibility making these binary options. We've discussed adding a (3rd) hybrid render option in the future for example. 
We can rename these using something other than "mode", just can't think of a good replacement.